storybooknative/src/types/StylesPropType.js

```diff
@@ -6,7 +6,11 @@
  */
 
 import {
-  type ____DangerouslyImpreciseStyleProp_Internal, // eslint-disable-line camelcase
+  type ____DangerouslyImpreciseStyle_Internal, // eslint-disable-line camelcase
 } from "react-native/Libraries/StyleSheet/StyleSheetTypes";
+import AnimatedNode from "react-native/Libraries/Animated/src/nodes/AnimatedNode";
 
-export type StylePropType = ____DangerouslyImpreciseStyleProp_Internal; // eslint-disable-line camelcase
+export type StylePropType = {
+  ...____DangerouslyImpreciseStyle_Internal, // eslint-disable-line camelcase
+  borderWidth?: number | AnimatedNode | string,
+};
```

Comment:
```
type ExactStyles = $Exact<____DangerouslyImpreciseStyle_Internal>;
type ShapedStyles = $Shape<ExactStyles>;

export type StylePropType = {|
  ...ShapedStyles,
  +borderWidth?: number | AnimatedNode | string,
|};
```
now we have exact and optional styles.

Reply:
However:
```
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ storybooknative/src/types/Style.js:14:6

Inexact ____DangerouslyImpreciseStyle_Internal [1] is incompatible with exact Style [2].

     11│ import AnimatedNode from "react-native/Libraries/Animated/src/nodes/AnimatedNode";
     12│
 [2] 13│ export type Style = {|
 [1] 14│   ...____DangerouslyImpreciseStyle_Internal,
     15│   +borderWidth?: number | AnimatedNode | string,
     16│ |};
     17│

```

Reply:
I think you need to stop and start flow server

Reply:
I read your comment too fast and forgot the first part 🥇